src/Operators/finitedifference.jl

```diff
@@ -1787,7 +1794,185 @@ Base.@propagate_inbounds function stencil_right_boundary(
     return Geometry.Contravariant3Vector(zero(eltype(eltype(A_field))))
 end
 
+######Limited Flux Methods######
+"""
+    U = TVDSlopeLimitedFlux(;boundaries)
+    U.(𝒜, Φ)
+𝒜, following the notation of Durran (Numerical Methods for Fluid
+Dynamics, 2ⁿᵈ ed.) is the antidiffusive flux given by
+𝒜 = ℱʰ - ℱˡ
+where h and l superscripts represent the high and lower order (monotone) 
+fluxes respectively. The effect of the TVD limiters is then to 
+adjust the flux 
+F_{j+1/2} = F^{1}_{j+1/2} + C_{j+1/2}(F^{h}_{j+1/2} - F^{l}_{j+1/2})
+where C_{j+1/2} is the multiplicative limiter which is a function of 
+the ratio of the slope of the solution across a cell interface.
+C=1 recovers the high order flux.
+C=0 recovers the low order flux. 
+
+Supported limiter types are 
+(1) RZeroLimiter (returns low order flux)
+(2) RHalfLimiter (the flux multiplier == 1/2)
+(3) RMaxLimiter (returns high order flux)
+(4) MinModLimiter
+(5) KorenLimiter
+(6) SuperbeeLimiter
+(7) MonotonizedCentralLimiter
+(8) VanLeerLimiter
+
+"""
+### Here we define abstract types and specific implementations 
+### TVD limiters (see, for instance, Durran's notes)
+abstract type AbstractTVDSlopeLimiter end
```

Comment:
Let's add this to the API, and move / delete the comments above so that the doc strict correctly attaches to `AbstractTVDSlopeLimiter`

Reply:
Updated.